client/landing/jetpack-cloud/sections/backups/backup-upsell/index.tsx

```diff
@@ -13,39 +13,70 @@ import DocumentHead from 'components/data/document-head';
 import Main from 'components/main';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import Upsell from 'landing/jetpack-cloud/components/upsell';
+import { UpsellComponentProps } from 'landing/jetpack-cloud/components/upsell-switch';
+import { recordTracksEvent } from 'state/analytics/actions';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const BackupsUpsellPage: FunctionComponent = () => {
+const BackupsVPActiveBody: FunctionComponent = () => {
 	const translate = useTranslate();
-	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const dispatch = useDispatch();
+	return (
+		<Upsell
+			headerText={ translate( 'Your site has VaultPress' ) }
+			bodyText={ translate(
+				'Your site already is backed up by VaultPress. You can find a link to your VaultPress dashboard below.'
```

Comment:
This is a small detail, but I think the copy here would read better in English as "Your site **is already** backed up by VaultPress." (Also applies to `scan/upsell.jsx`)

Reply:
Nice catch!

Reply:
Resolved in 6341e6d.